### PR TITLE
Add documents for CEL APIs

### PR DESCRIPTION
**Pull request by goccy**
SSIA

### CODE REVIEW

File: docs/cel/time.md [hunk 1]
```diff
@@ -0,0 +1,1187 @@
...
+
+The API for this package was created based on Go's [`time`](https://pkg.go.dev/time) package.
+
+# Index
+
+## Constants
+
+- [`LAYOUT`](#layout)
+- [`ANSIC`](#ansic)
+- [`UNIX_DATE`](#unix_date)
```
Comment (shuheiktgw) [inline docs/cel/time.md:11]: At least in my local environment, the path with an underscore does not work... `#unixdate` seems to work fine!

Comment (goccy) [reply docs/cel/time.md:11]: It appears to be working
https://github.com/mercari/grpc-federation/blob/cel-docs/docs/cel/time.md#unix_date

File: docs/cel/time.md [hunk 2]
```diff
@@ -0,0 +1,1187 @@
...
+
+### Examples
+
+```cel
+grpc.federation.time.toDuration(grpc.federation.time.HOUR + 2*grpc.federatino.time.MINUTE).string() //=> 1h2m
+```
+
+## Duration.truncate
+
+`truncate` returns the result of rounding d toward zero to a multiple of `m`. If `m` <= 0, Truncate returns `Duration` unchanged.
```
Comment (shuheiktgw) [inline docs/cel/time.md:604]:
```suggestion
`truncate` returns the result of rounding d toward zero to a multiple of `m`. If `m` <= 0, `truncate` returns `Duration` unchanged.
```

nit

**Review by shuheiktgw — APPROVED** [review]
Left some minor comments but LGTM 🙆‍♂️

**Pull request merged**